### ISSUE TITLE
Document Binary Format for Loader Configuration

### ISSUE THREAD

**Issue by Sewer56** [Reloaded-Project/Reloaded-III#34]
Copied from [Memory Limits](https://github.com/Reloaded-Project/Reloaded-III/issues/13)

## Binary format for Loader Config

Runtime information for loader such as:
- Which mods to load
- Mod locations
- Mod metadata

Will need to use a binary file to save on code size.

This is planned to reduce startup times, as a merged file is needed to reduce I/O overhead.
So no special accommodations needed here.

Basically gather what info from the loadout the loader needs to know based on the existing spec,
serialize with rkyv, and deserialize on the other end.

**Comment by Sewer56** [comment]
## [Loader] 'Activate' Button to set a loadout to 'Active' 

This writes a file to the game's folder.
The only game folder edit allowed (aside from installing loader).
Said file lists the path to the currently active loadout.
